lib/checkCount.js

```diff
@@ -0,0 +1,10 @@
+exports.PRCount = function (response) {
+    let countIssues = 0;
+    // Check for issues that are not also PRs
+    for (let i = 0; i < response.data.length; i++) {
```

Comment:
Could we use `Array.filter()` here and just the total number of issues so we don't have to loop though the data?